remote/advancedcontainers/docker-options.md

```diff
@@ -0,0 +1,112 @@
+---
+Order: 16
+Area: advancedcontainers
+TOCTitle: Docker Options
+PageTitle: Remote - Containers Docker options
+ContentId:
+MetaDescription: Remote - Containers Docker options
+DateApproved: 2/4/2022
+---
+
+# Alternate ways to install Docker
+
+You may harness Docker in the Remote - Containers extension in a few ways:
+* Docker installed locally
+* Docker installed on another machine or remote environment
+* Other Docker compliant CLIs, installed locally or in a remote environment
+     * Remote - Containers interacts with CLIs; it makes no assumptions about how a container engine works and does not interact with container engines or daemon directly.
+     * Please note that other Docker compliant CLIs are not officially supported.
+
+Continue reading to learn alternate ways you may install and leverage Docker or a Docker compliant CLI.
+
+## Podman
+
+[Podman](https://podman.io/) 1.9+ is mostly compatible with Docker's CLI commands and therefore generally does work if you update the `Docker Path` setting (which may be accessed through **Remote > Containers: Docker Path**) to `podman` on Linux.
+
+![Docker Path setting](images/platform-options/docker-path-setting.png)
+
+However, certain tricks like [Docker-from-Docker do not work](https://github.com/containers/libpod/issues/4056#issuecomment-535511841) due to limitations in Podman. This affects the **Remote-Containers: Try a Development Container Sample...** and **[Remote- Containers: Clone Repository in Container Volume...](/docs/remote/containers.md#quick-start-open-a-git-repository-or-github-pr-in-an-isolated-container-volume)** commands.
+
+Docker Compose is also not supported by Podman.
```

Comment:
There used to be Podman Compose, not sure about its status.

Reply:
Found it: https://github.com/containers/podman-compose. Looks like it was last updated 9 days ago.

From [an article](https://crunchtools.com/should-i-use-docker-compose-or-podman-compose-with-podman/) a few months ago:

`The podman-compose utility is part of the github.com/containers project on GitHub and aligns itself closely with Podman. It was first compose-tool to work rootless. It interacts with the Podman CLI, partially because it was developed before the API, but more importantly because the fork/exec model offers some advantages over using a REST based API. For example, imagine you are compiling he Linux kernel in a container. With the fork/exec model you can use all of the data from the local directory, but with the API, you have to copy the context directory across the socket to do the build. The podman-compose community tests podman-compose, but it does not [appear](https://github.com/containers/podman-compose/pull/339) to have CI/CD.`

Sounds like linking to Podman Compose is reasonable?

Reply:
Sounds good 👍